RecaptchaEnterprise/RecaptchaInterop/Public/RecaptchaInterop/RCARecaptchaProtocol.h

```diff
@@ -58,6 +58,19 @@ NS_ASSUME_NONNULL_BEGIN
                                        NSError *_Nullable error))completion
     NS_SWIFT_NAME(getClient(withSiteKey:completion:));
 
+/**
+ * Builds a new reCAPTCHA Client for the given Site Key.
+ *
+ * The SDK currently supports one Site Key. Passing a different Site Key will
+ * throw an exception.
+ *
+ * @param siteKey reCAPTCHA Site Key for the app.
+ * @param completion Callback function to return the RecaptchaClient or an error.
+ */
++ (void)fetchClientWithSiteKey:(nonnull NSString *)siteKey
```

Comment:
optional – add a note in the doc comment as to why this method is preferred over the old one

And a more general question: could the implementation of getClientWithSiteKey be change to that of the propsoed fetchClientWithSiteKey?

Reply:
As we discussed getClient was removed from the implementation as it will be removed in future versions of the SDK.